cgi-bin/LJ/Talk.pm

```diff
@@ -3695,7 +3695,13 @@ sub make_preview {
             unless $_ eq 'body' || $_ eq 'subject' || $_ eq 'prop_opt_preformatted' || $_ eq 'editreason';
     }
 
-    $ret .= "<br /><input type='submit' value='$BML::ML{'/talkpost_do.bml.preview.postcomment'}' />\n";
+    my $post_disabled = $u->does_not_allow_comments_from($remote)
+        || ($u->{'opt_whocanreply'} eq 'reg' && $remote->is_identity && ! ( $remote->is_validated || $u->trusts( $remote )));
```

Comment:
@zorkian Folding it into that method would disable Quick Reply entirely in this situation due to S2.pm:3677 and I think removing that line would require more comprehensive testing and more thought in the area of permission to comment generally.... which @swaldman3 seems to be planning under #1704 and #1661, so perhaps it's best to wait for those to land before worrying about this PR.